scripts/kani-perf.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+if [[ -z $KANI_REGRESSION_KEEP_GOING ]]; then
+  set -o errexit
+fi
+set -o pipefail
+set -o nounset
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+KANI_DIR=$SCRIPT_DIR/..
+
+# Build all packages in the workspace
+cargo build --workspace
+
+PERF_DIR="${KANI_DIR}/tests/perf"
+
+# Copy expected files from overlay directories
+to_delete=
+for overlay_dir in ${PERF_DIR}/overlays/*/; do
+  orig_dir=$(basename ${overlay_dir})
+  echo "Copying overlays for $orig_dir"
+  copy_output=$(cp -r -v ${overlay_dir}* ${PERF_DIR}/${orig_dir}/)
+  copied_files=$(echo ${copy_output} | rev | cut -d' ' -f 1 | rev | tr -d "'")
+  # Add to the list of files to delete
+  to_delete="${to_delete} ${copied_files}"
+done
+
+suite="perf"
+mode="cargo-kani"
+echo "Check compiletest suite=$suite mode=$mode"
+cargo run -p compiletest --quiet -- --suite $suite --mode $mode --quiet
```

Comment:
I don't think we need `--quiet` here.

Reply:
Now it's at least showing runtime for the whole suite, which is better than nothing :D